python_modules/dagster/dagster_tests/definitions_tests/auto_materialize_tests/test_asset_daemon_fast.py

```diff
@@ -29,6 +22,12 @@
 )
 from .scenarios.scenarios import ASSET_RECONCILIATION_SCENARIOS
 
+#############################
+# FAST auto materialize tests
+#############################
+#
+# Run the auto materialize scenarios, but use an InternalAssetGraph instead of External to speed things up.
+
```

Comment:
this suite is 30s vs 2.5 min for the daemon suite, so its worth keeping this one around